app/views/services/_mobile_menu.html.erb

```diff
@@ -8,7 +8,12 @@
           <div class="bar2"></div>
           <div class="bar3"></div>
         </span>
-        <span id="mobile-navbar-brand" class="navbar-brand">See More</span>
+        <span id="mobile-navbar-brand" class="navbar-brand">
```

Comment:
This seems to exactly duplicate what's above. Could be refactored into a partial or a helper. Something like:
```
def menu_category_list(categories)
  categories.collect(&:label).join(" and ")
end

```

Reply:
helper. may need a hand on how to test.

Reply:
Testing would look like
- create 2 categories with factories
- create another entity
- add the 2 categories to the entity

and then something like:
`expect(helpers.menu_category_list(entity.categories)).to eql "Cat1 and Cat2"`

You probably want to test that it does what yu expect with a single category as well.

https://github.com/tulibraries/manifold/blob/master/spec/helpers/route_helpers_spec.rb#L8-L13

Reply:
@cdoyle-temple :point_up:

Reply:
Rspec test is done, Travis checks have passed.